string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,16 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  var normalArr = [];
+  var backArr = [];
+  var backText = '';
+  for (var i = 0; i < word.length; i++) {
+    normalArr.push(word[i]);
+  }
+  for (var k = word.length - 1; k >= 0; k--) {
+    backArr.push(normalArr[k]);
+  }
+  for (var t = 0; t < word.length; t++) {
+    backText += backArr[t];
+  }
```

Comment:
You can accomplish this using just 1 loop, rather than breaking it up into 3 loops
```
function reverseWord(word) {
  var reversed = '';
  for (var i = word.length - 1; i >= 0; i--) {
    reversed += word[i];
  }
  return reversed;
}
```